.janitor/dspace-update.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM janitortechnology/dspace
```

Comment:
What is the purpose of this Dockerfile?  How does it relate to the other file?

Since this file is performing a git fetch within the image, does this file need to reside in DSpace-angular?

Reply:
I am not sure, but I think it would be a good idea to keep everything together.

Reply:
Such an `*-update.dockerfile` can be used to build incremental image updates (as opposed to full image rebuilds, which may take more disk space). This technique was used to upgrade project images while saving disk space in our servers, but it is not very useful for relatively small projects (image < 30GB).

TL;DR: Feel free to remove this file entirely.